src/dotnet-test-xunit/project.json

```diff
@@ -17,25 +17,30 @@
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
-        "dnxcore50",
+        "dotnet",
```

Comment:
`xunit.runner.reporter` 2.1.0 targets `dotnet. `Newtonsoft.Json` targets `portable-dnxcore50+net45+win8+wp8+wpa81` and DiaSymReader targets `portable-net45+win8`. Explicitly listing so we know we're getting the exact packages we want.